src/translation/translation_pl_PL.ts

```diff
@@ -863,7 +863,7 @@
     <message>
         <location filename="../clientdlg.cpp" line="153"/>
         <source>If this LED indicator turns red, you will not have much fun using %1.</source>
-        <translation type="unfinished"></translation>
+        <translation>Jeśli ta dioda zmieni kolor na czerwony, korzystanie z aplikacji %1 może być utrudnione.</translation>
```

Comment:
If aplikacji is application, we've tried to get rid of "Jamulus Software" in English. So probably if it also sounds natural without, it would be best to not have software or application in the text. Just plain %1 standing for Jamulus

Reply:
But it quite correct in Polish language and not pompous at all like "Jamulus Software" may sound like in English.

Reply:
Good! Thanks